packages/suite/src/middlewares/wallet/coinjoinMiddleware.ts

```diff
@@ -116,9 +117,15 @@ export const coinjoinMiddleware =
             }
         }
 
-        if (action.type === DISCOVERY.START || blockchainActions.synced.match(action)) {
+        if (
+            action.type === discoveryActions.startDiscovery.type ||
+            blockchainActions.synced.match(action)
```

Comment:
nitpick: `isAnyOf` can be used

Reply:
[a802d5f](https://github.com/trezor/trezor-suite/pull/8616/commits/a802d5f07e68c6142a2e8f18da68635a240d397e)